components/resources/brave_components_resources.grd

```diff
@@ -69,6 +69,7 @@
       <part file="speedreader_resources.grdp" />
       <part file="brave_flags_ui_resources.grdp" />
       <part file="ipfs_resources.grdp" />
+      <part file="trezor_bridge_resources.grdp" />
```

Comment:
This is the one I think we should add in the trezor-bridge component, not this shared component, please

Reply:
we can do this as a follow-up since I see we're still adding individual component resources to this shared .grd file, e.g. wallet and vpn. However, I think you might need to wrap with a condition, e.g. 
```xml
<if expr="not is_android"
```
?

Reply:
done